spec/models/page_spec.rb

```diff
@@ -6,7 +6,9 @@
   describe "#processed_html" do
     it "accepts body markdown and turns it into html" do
       page.body_markdown = "Hello `heyhey`"
-      page.save
+      perform_enqueued_jobs do
```

Comment:
Same as above - processing html is not done asynchronously, so there's no need to run the jobs. You can add another test to check that the job was enqueued.

Reply:
I did that.